api/02-writing-ext-themes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Writing Extension and Themes
+---
+
+This document outlines how to write your own extensions and themes for Phoenix Code.
+
+## Creating an extension
+Click on one of the links below and follow the instructions there to start:
+* [Create a theme](https://github.com/phcode-dev/theme-template)
+* [Create an extension](https://github.com/phcode-dev/extension-template)
+* [Create a node.js extension](https://github.com/phcode-dev/extension-node-template)
+
+## API docs
+Please refer to the links below for extension API docs and how to's.
+* Phoenix Code APIs - [API Reference](./API-Reference/NodeConnector)
+* Common How-Tos: https://github.com/brackets-cont/brackets/wiki/How-to-Write-Extensions#common-how-tos
```

Comment:
We should deprecate this page in another pull request and move that brackets wiki contents to a suitable location in our docs. And remove things in that dos that doesnt make sense for phoenix code extension dev.